sw/applications/example_dma_sdk/main.c

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 EPFL and Politecnico di Torino.
+// Solderpad Hardware License, Version 2.1, see LICENSE.md for details.
+// SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+//
+// File: dma_sdk.c
+// Author: Juan Sapriza
```

Comment:
maybe can remove Polito from the licence - also, copyright 2024 would be nicer

Reply:
I updated the copyright year. Given that the SDK was written in Polito, I feel it's a shared work. 
Is this relevant in any situation tho? jahaha